packages/beacon-node/src/execution/builder/http.ts

```diff
@@ -59,6 +58,7 @@ export class ExecutionBuilderHttp implements IExecutionBuilder {
       },
       {config, metrics: metrics?.builderHttpClient}
     );
+    logger?.info("External builder", {url: toSafePrintableUrl(baseUrl)});
```

Comment:
Builder is a bit of an outlier right now, we allow to set multiple URLs via `--builder.urls` but only use the first URL (`opts.urls[0]`). I opted to only log the URL which is actually used.

@g11tech Is it intentional that we don't allow a fallback for the builder? I guess mev-boost is already doing the multiplexing and builder flow is stateful which makes fallback logic a bit problematic.

Maybe we want to reflect that in the CLI as well as currently it looks like it supports multiple urls

![image](https://github.com/ChainSafe/lodestar/assets/38436224/3feb859b-0374-4e33-afc5-df38029f36d7)